app/src/main/java/com/example/aplikacja/AddCarFragment.kt

```diff
@@ -0,0 +1,129 @@
+package com.example.aplikacja
+
+import android.app.AlertDialog
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.Spinner
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import java.io.FileWriter
+
+
+/**
+ * A simple [Fragment] subclass as the second destination in the navigation.
+ */
+class AddCarFragment : Fragment() {
+    private val selectTypeSpinner = view?.findViewById<Spinner>(R.id.selectTypeSpinner)
+    private val selectBrandSpinner = view?.findViewById<Spinner>(R.id.selectBrandSpinner)
+    private val selectModelSpinner = view?.findViewById<Spinner>(R.id.selectModelSpinner)
+    private val addLicensePlateInput = view?.findViewById<EditText>(R.id.addLicensePlateInput)
+    private val addMeterStatusInput = view?.findViewById<EditText>(R.id.addMeterStatusInput)
+    private val warningText = view?.findViewById<TextView>(R.id.secondScreen_warning)
+    private val addFirstCarTitleText = view?.findViewById<TextView>(R.id.addFirstCarTitle)
+
+
+    override fun onCreateView(
+            inflater: LayoutInflater, container: ViewGroup?,
+            savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.addcar_screen, container, false)
+
+
+        loadData()
+
+    }
+
+    private fun loadData(){
+        val shearedPreferences = requireActivity().getSharedPreferences("sheredPrefs", Context.MODE_PRIVATE)
+        val savedName = shearedPreferences?.getString("NAME_KEY", null)
+
+        //addFirstCarTitleText?.text = savedName
```

Comment:
czy ten zakomentowany kod jest istotny, albo przyda się w niedalekiej przyszłości?
jeżeli nie to lepiej go usunąć